tests/test_plugin_specs.lua

```
@@ -0,0 +1,331 @@
+  package.loaded["jj.diff"] = nil
+  package.preload["jj.diff"] = function()
+    return { open_vdiff = function() end, open_hsplit = function() end }
+  end
+
+  package.loaded["jj.picker"] = nil
+  package.preload["jj.picker"] = function()
+    return { status = function() end, file_history = function() end }
+  end
+
+  -- Snapshot pre-existing mappings so we only assert on what we added.
+  local pre = {}
```

Luacheck (1.2.0)

[warning] variable 'pre' is mutated but never accessed

(W241)